lnbits/core/views/generic.py

```diff
@@ -149,8 +151,8 @@ async def wallet(
         )
 
     logger.debug(f"Access wallet {wallet_name}{'of user '+ user.id if user else ''}")
-    wallet = user.get_wallet(wallet_id)
-    if not wallet:
+    userwallet = user.get_wallet(wallet_id)
```

Comment:
Why rename variable?

Reply:
if not mypy complains ```lnbits/core/views/generic.py:154: error: Incompatible types in assignment (expression has type "Optional[Wallet]", variable has type "Wallet")```
i guess ```create_wallet``` above return Optional[Wallet] and ```user.get_wallet()``` return Wallet, mypy doesnt like reusing same variable with different types.